app/controllers/taxi_requests_controller.rb

```diff
@@ -3,6 +3,12 @@
 class TaxiRequestsController < ApplicationController
 
   def index
-    # 인증 인가 테스트를 위한 골격 구현 코드
+    taxi_requests = if current_user.user_type.eql? 'passenger'
```

Comment:
enum 타입을 활용하시면 좋을 것 같습니다.

```suggestion
    taxi_requests = if current_user.user_type.eql? User.user_types[:passenger]
```

Reply:
넵 반영하였습니다 :)